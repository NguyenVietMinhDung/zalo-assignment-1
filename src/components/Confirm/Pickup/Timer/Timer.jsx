import React from 'react';
import { Row, Col } from 'react-grid-system';
import './Timer.scss';

const Timer = () => (
  <Row>
    <Col className="time" xs={12}>
      <div className="time__icon">
        <i className="lni lni-16 lni-timer" />
      </div>
      <div className="time__value">
        ASAP - 15:30, HÃ´m nay 30/06
      </div>
      <div className="time__modify-btn">
        <i className="lni lni-sm lni-chevron-right" />
      </div>
    </Col>
  </Row>
);

export default Timer;
