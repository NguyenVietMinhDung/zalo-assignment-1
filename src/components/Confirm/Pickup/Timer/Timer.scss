@import '../../../../styles/variables';

.time {
	display: flex;

	padding-top: 15px;
	padding-bottom: 15px;

	border-bottom: 1px solid $white-smoke;

	&__icon {
		margin-right: 5px;

		i {
			opacity: 0.2;

			font-weight: bold;
		}
	}

	&__value {
		flex: 1;
	}

	&__modify-btn {
		cursor: pointer;
	}
}
