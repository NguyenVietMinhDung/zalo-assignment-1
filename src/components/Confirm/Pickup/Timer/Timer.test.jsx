import React from 'react';
import { shallow } from 'enzyme';
import Timer from './Timer';

describe('<Timer />', () => {
  test('renders correctly', () => {
    const component = shallow(<Timer />);
    expect(component).toMatchSnapshot();
  });
});
