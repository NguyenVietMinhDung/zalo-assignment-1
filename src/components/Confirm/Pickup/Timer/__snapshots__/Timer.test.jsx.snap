// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Timer /> renders correctly 1`] = `
<Row
  align="normal"
  component="div"
  debug={false}
  gutterWidth={null}
  justify="start"
  nogutter={false}
  nowrap={false}
  style={Object {}}
>
  <Col
    className="time"
    component="div"
    debug={false}
    lg={null}
    md={null}
    offset={Object {}}
    pull={Object {}}
    push={Object {}}
    sm={null}
    style={Object {}}
    width={null}
    xl={null}
    xs={12}
    xxl={null}
  >
    <div
      className="time__icon"
    >
      <i
        className="lni lni-16 lni-timer"
      />
    </div>
    <div
      className="time__value"
    >
      ASAP - 15:30, HÃ´m nay 30/06
    </div>
    <div
      className="time__modify-btn"
    >
      <i
        className="lni lni-sm lni-chevron-right"
      />
    </div>
  </Col>
</Row>
`;
